### DISCUSSION TITLE
How to start trial period over?

### DISCUSSION THREAD

**Discussion by mattseaboard** [solidify/scale#29] [Q&A] [answered]
Hi there,  We installed this extension a while ago but never got around to testing it out.  Our new scrum master is now wanting to test it out but our trial is expired.  Is there a way to reset the trial period?

**Comment by JimmyKB** [comment, marked as answer]
Hello @mattseaboard Please contact us at Scale@solidify.dev and write your organization/email and we can give you a shorter prolonged trial.

Kind regards

**Comment by mattseaboard** [comment]
Hi @JimmyKB, I did do that and they were able to reset my trial, thanks!
